bap-llvm.opam.template

```diff
@@ -3,6 +3,7 @@ build: [
   [
   "ocaml" "tools/configure.ml"
   "--with-llvm-config=%{conf-bap-llvm:config}%"
+  "--%{llvm-shared?disable:enable}%-llvm-static"
```

Comment:
wow, didn't know about this syntax, is it new? (I mean will it work with opam 2.0)

Reply:
Yes, this was added in opam 1.2.1.